portfolio/src/main/webapp/script.js

```diff
@@ -29,3 +29,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerHTML = gif;
 }
+
+/**
+ * gets title from server
+ */
+function getTitle() {
+  console.log('in getTitle() function');
```

Comment:
This line looks like testing code, is it intended?

Reply:
Yes, so sorry, I forgot to delete them. I have deleted them in the new commit below! :)